Source/Extensions/Blazorise.Components/RouterTabsPageAttribute.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Blazorise.Components;
+/// <summary>
+/// Attribute that sets the Router Tabs Page attributes
+/// </summary>
+public class RouterTabsPageAttribute : Attribute
+{
+    /// <summary>
+    /// Sets the title of the router tab.
+    /// </summary>
+    public string Title { get; set; }
+
+    /// <summary>
+    /// Sets the css class of the router tab.
+    /// </summary>
+    public string TabCssClass { get; set; }
```

Comment:
Why are `CssClass`es needed?

Reply:
Why not? It's an attribute, user sets whatever he wants in there?